src/components/SoundBox/ColumnOfButtons/ColumnOfButtons.js

```diff
@@ -12,110 +13,64 @@ export const ColumnOfButtonsWrapper = styled.div`
 export const columnReducer = (state, action) => {
   switch (action.type) {
     case 'TOGGLE_SELECTED_BOX': {
-      const { clickedButton } = action;
-      return clickedButton === state ? false : clickedButton;
+      const { clickedButton, soundBank, audioFiles, clock, audioType } = action;
+      const { selectedButton, getEvent } = state;
+      if (getEvent) {
```

Comment:
I also recommend extracting this code to separate method and move this event staff logic to action and then in reducer just only change the state.